libs/eventually-broker/src/cluster/state.ts

```diff
@@ -286,7 +286,7 @@ export const state = singleton(function state(): State {
     const message = `Worker ${workerId} exited with ${
       signal ? signal : `code ${code}`
     }`;
-    log().info("bgRed", `[${process.pid}]`, message);
+    log().info("bgRed", `[${process?.pid || ''}]`, message);
```

Comment:
Arguably this does not make sense... since process is a global nodejs thing... but that is what it is complaining about when dying as seen in the link.

It would be interesting to try and replicate this error locally somehow.

https://us5.datadoghq.com/logs?query=env%3Aproduction%20service%3Abroker%20status%3Aerror%20&agg_m=count&agg_m_source=base&agg_t=count&cols=host%2Cservice&event=AgAAAZIHZvqbD638bgAAAAAAAAAYAAAAAEFaSUhad0dtQUFBQnZYMFpXd3FjNkFBTAAAACQAAAAAMDE5MjA3NjctMjM5ZS00NWZmLTkxNDYtZjM4NDQ1MDAxMzYw&fromUser=true&messageDisplay=inline&refresh_mode=sliding&storage=hot&stream_sort=desc&viz=stream&from_ts=1726699326930&to_ts=1726701102380&live=false